test/position-managers/PositionManager.multicall.t.sol

```diff
@@ -71,4 +102,111 @@ contract PositionManagerMulticallTest is Test, PosmTestSetup, LiquidityFuzzers {
         assertEq(result.amount0(), amountSpecified);
         assertGt(result.amount1(), 0);
     }
+
+    function test_multicall_permit_mint() public {
+        PositionConfig memory config = PositionConfig({
+            poolKey: key,
+            tickLower: TickMath.minUsableTick(key.tickSpacing),
+            tickUpper: TickMath.maxUsableTick(key.tickSpacing)
+        });
+        // 1. revoke the auto permit we give to posm for 1 token
+        vm.prank(bob);
+        permit2.approve(Currency.unwrap(currency0), address(lpm), 0, 0);
+
+        (uint160 _amount,, uint48 _expiration) =
+            permit2.allowance(address(bob), Currency.unwrap(currency0), address(this));
+
+        assertEq(_amount, 0);
+        assertEq(_expiration, 0);
+
+        uint256 tokenId = lpm.nextTokenId();
+        bytes memory mintCall = getMintEncoded(config, 10e18, bob, ZERO_BYTES);
+
+        // 2 . call a mint that reverts because position manager doesn't have permission on permit2
+        vm.expectRevert(abi.encodeWithSelector(IAllowanceTransfer.InsufficientAllowance.selector, 0));
+        vm.prank(bob);
+        lpm.modifyLiquidities(mintCall, _deadline);
+
+        // 3. encode a permit for that revoked token
+        IAllowanceTransfer.PermitSingle memory permit =
+            defaultERC20PermitAllowance(Currency.unwrap(currency0), permitAmount, permitExpiration, permitNonce);
+        permit.spender = address(lpm);
+        bytes memory sig = getPermitSignature(permit, bobPrivateKey, PERMIT2_DOMAIN_SEPARATOR);
+
+        bytes[] memory calls = new bytes[](2);
+        calls[0] = abi.encodeWithSelector(Permit2Forwarder(lpm).permit.selector, bob, permit, sig);
```

Comment:
why isnt this just `Permit2Forwarder.permit.selector` ? why does it need a parameter?

Reply:
Oh i can do that!

Reply:
i really wanted it to be possible to always do IPositionManager._.selector  but we cant do that bc not all external functions are defined on IPositionManager

so then I tried lpm.selector but that also doesnt work.. its super weird


if u know how we can fix this so we dont always have to use the specific contract that would be the best but fixed for now like this